src/client/components/Product/Product.component.js

```diff
@@ -15,13 +17,24 @@ const heart = {
 };
 
 export default function Product({
+  id,
   image,
   name,
   price,
   currency,
   onClick,
-  addToFavorites,
+  
 }) {
+
+  
+  const addToFavorites=()=>{
+    console.log('in add to favorites');
+    addProductToFavorites(10,id);
```

Comment:
addProductToFavorites(10,id) can be given directly in the line 68, onClick!

Reply:
yes can be done but I think this way is also ok